src/sidebar/panel.js

```diff
@@ -218,7 +219,7 @@ quill.on("text-change", (delta, oldDelta, source) => {
   });
 });
 
-const enableSync = document.getElementById('enableSync');
+const enableSync = document.getElementById('enable-sync');
```

Comment:
Changed to hyphenated to be consistent with options.html: `<span id="redirect-url"></span>`